opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -2307,6 +2324,14 @@ std::vector<std::string> FieldProps::fip_regions() const
     return result;
 }
 
+void FieldProps::set_active_indices(const std::vector<int>& indices)
+{
+    std::size_t idx = 0;
```

Comment:
May want to call `m_active_index.clear()` ahead of the loop here.  Otherwise, we risk inconsistent behaviour if someone calls `set_active_indices()` twice.

Reply:
yeah, will add